AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/README.md

```diff
@@ -124,43 +117,10 @@ conda install jupyter nb_conda_kernels
 4. Change the kernel to **Python [conda env:tensorflow]**.
 5. Click the **Run** button to move through the cells in sequence.
 
-### Run the Sample on Intel® DevCloud (Optional)
-
-1. If you do not already have an account, request an Intel® DevCloud account at [*Create an Intel® DevCloud Account*](https://intelsoftwaresites.secure.force.com/DevCloud/oneapi).
-2. On a Linux* system, open a terminal.
-3. SSH into Intel® DevCloud.
-   ```
-   ssh DevCloud
-   ```
-   > **Note**: You can find information about configuring your Linux system and connecting to Intel DevCloud at Intel® DevCloud for oneAPI [Get Started](https://devcloud.intel.com/oneapi/get_started).
-
-4. You can specify a CPU node using a single line script.
-   ```
-   qsub  -I  -l nodes=1:xeon:ppn=2 -d .
-   ```
-
-   - `-I` (upper case I) requests an interactive session.
-   - `-l nodes=1:xeon:ppn=2` (lower case L) assigns one full GPU node.
-   - `-d .` makes the current folder as the working directory for the task.
-
-     |Available Nodes |Command Options
-     |:---            |:---
-     |GPU	            |`qsub -l nodes=1:gpu:ppn=2 -d .`
-     |CPU	            |`qsub -l nodes=1:xeon:ppn=2 -d .`
-
-5. Activate conda. 
-` $ conda activate`
-6. Follow the instructions to open the URL with the token in your browser.
-7. Locate and select the Notebook.
-   ```
-   ResNet50_Inference.ipynb
-   ````
-8. Change the kernel to **Python [conda env:tensorflow]**.
-9. Run every cell in the Notebook in sequence.
 
 ## License
 
 Code samples are licensed under the MIT license. See
 [License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
 
-Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt).
\ No newline at end of file
+Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt).
```

Comment:
At the end of the file we need to insert:
*Other names and brands may be claimed as the property of others. [Trademarks](http://www.intel.com/content/www/us/en/legal/trademarks.html)

This request is relevant for any README file where we use asterisks

Reply:
addressed it accordingly